bigquery-antipattern-recognition/src/main/java/com/google/zetasql/toolkit/antipattern/util/BigQueryHelper.java

```diff
@@ -137,8 +137,7 @@ private TableResult getQueriesFromIS(
             + "  AND total_slot_ms > "
             + slotsMsMin
             + "\n"
-            + "  AND (statement_type != \"SCRIPT\" OR statement_type IS NULL)\n"
-            + "  AND (reservation_id != 'default-pipeline' or reservation_id IS NULL)\n"
+            + "  AND reservation_id != 'default-pipeline' \n"
```

Comment:
@franklinWhaite Removed this portion to include scripts in information schema queries